packages/molecules/src/navbar/index.tsx

```
@@ -64,28 +113,29 @@
-            {config.logos.showCenterLogos &&
-              config.logos.centerLogoIcons.map((logo: { id: string; src: string }) => (
+            {centerLogoIcons &&
+              centerLogoIcons.map((logo: { id: string; src: string }) => (
                 <img
                   key={logo.id}
                   src={logo.src}
                   alt={`Logo ${logo.id}`}
-                  style={{ maxHeight: '48px' }}
+                  style={{ maxHeight: '60px' }}
                 />
               ))}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)